### PR TITLE
Add 2 US references

### PR DESCRIPTION
**Pull request by fbridault**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +56,10 @@ A curated list of awesome Direct Volume Rendering articles, software and resourc
 
 ### Ultrasound
 
-- **The ultrasound visualization pipeline** (2014), Å. Birkeland [pdf](https://pdfs.semanticscholar.org/b713/ccf2c88e73e951489cc89ecc1381a7e425fd.pdf)
+- **The ultrasound visualization pipeline** (2014), Å. Birkeland et al. [pdf](https://pdfs.semanticscholar.org/b713/ccf2c88e73e951489cc89ecc1381a7e425fd.pdf)
+- **Feature Emphasis and Contextual Cutaways for Multimodal Medical Visualization** (2007), M. Burns et al.  [pdf](https://gfx.cs.princeton.edu/pubs/Burns_2007_FEA/contextual_cutaways.pdf)
+- **New flexible multi-volume rendering technique for ultrasound imaging. Medical Imaging** (2009), E.H. Kim et al.
+[Research Gate](https://www.researchgate.net/publication/252244444_New_flexible_multi-volume_rendering_technique_for_ultrasound_imaging)
```
Comment (gKevin) [inline README.md]: I don't find it awesome to include articles that aren't freely available, it goes against the awesome philosophy imo.

Comment (fbridault) [reply README.md]: That would be a bit restrictive... People can find the article if they have a paid subscription by their company or institution, or at worst they can also find it on scihub..

Comment (gKevin) [reply README.md]: Ok, but I believe that something should be either educational, practical or somewhat ground-breaking in order to be listed here. More so if it isn't easily accessible. After all, this should be a curation and not a collection.

The article presented here isn't particularly awe-inspiring to me. The technique presented here basically suggests to fuse both volumes (using a threshold based function) and render the fused result. I'm not against adding this if it is the latest and best technique on the matter but I kind of doubt that.

Comment (fbridault) [reply README.md]: Agreed that makes sense. I removed it and I propose you instead a nice Phd Thesis from 2016, that covers not only volume visualisation but also volume reconstruction, with a great bibliography.

Comment (aancel) [reply README.md]: I think that we can thus resolve this conversation !    
Feel free to re-open a comment or issue if needed @gKevin

**Comment by aancel** [comment]
There is just one thing that is bugging me:     
Is there a reason why the publications are not sorted by publication year ?

File: README.md [hunk 2]
```diff
@@ -56,7 +56,9 @@ A curated list of awesome Direct Volume Rendering articles, software and resourc
 
 ### Ultrasound
 
-- **The ultrasound visualization pipeline** (2014), Å. Birkeland [pdf](https://pdfs.semanticscholar.org/b713/ccf2c88e73e951489cc89ecc1381a7e425fd.pdf)
+- **The ultrasound visualization pipeline** (2014), Å. Birkeland et al. [pdf](https://pdfs.semanticscholar.org/b713/ccf2c88e73e951489cc89ecc1381a7e425fd.pdf)
+- **Real-Time Processing for Advanced Ultrasound Visualization** (2016), Schulte zu Berge, Christian Ulrich, Nassir Navab, Nassir Navab, and Bernhard Preim. [pdf](https://mediatum.ub.tum.de/doc/1285497/file.pdf) 
+- **Feature Emphasis and Contextual Cutaways for Multimodal Medical Visualization** (2007), M. Burns et al. [pdf](https://gfx.cs.princeton.edu/pubs/Burns_2007_FEA/contextual_cutaways.pdf)
```
Comment (aancel) [inline README.md:61]: There is just one thing that is bugging me:
Is there a reason why the publications are not sorted by publication year ?

**Review by aancel — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (fbridault) [reply README.md:61]: Nope, that's a good suggestion. More recent first I guess ?

Comment (aancel) [reply README.md:61]: It seems that generally least recent first is more used in the current list (There are some mixup at some point) :smile:

Comment (fbridault) [reply README.md:61]: It really think it makes more sense to sort from most recent to least recent... I applied that on all publications.

**Pull request merged**